packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1783,6 +1788,10 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
                 0x12e /* "Container Context should already be in attached state" */);
             this.emit("attached");
         }
+
+        if (attachState === AttachState.Attached && !this.pendingStateManager.hasPendingMessages()) {
```

Comment:
We don't care about the local state passed to the pending state manager here?

Reply:
I have that note here: https://github.com/microsoft/FluidFramework/issues/8893. I'll sort out the pending (manager) ticket as a follow up in Feb. However, realistically with use cases we have, this is a minor issue at this point.